eslzArm/eslzArm.json

```diff
@@ -598,6 +598,7 @@
             "managementGroups": "[uri(deployment().properties.templateLink.uri, 'managementGroupTemplates/mgmtGroupStructure/mgmtGroups.json')]",
```

Comment:
In this file we are still missing the input parameters as well as the deployment of the assignment, correct?

Reply:
This is the deployment of the dataPolicies.json (definitions) and the parameter for eslz prefix is provided. The assignment are separate deployments.